presentation/src/main/java/com/knocklock/presentation/lockscreen/LockScreenNotiItem.kt

```diff
@@ -35,44 +42,118 @@ data class Notification(
+        clickableState = notificationList.size >= 2
+    }
+
+    Column(
+        modifier = Modifier.clickable(enabled = clickableState) {
+            expandState = !expandState
+        }
+    ) {
+        Box {
+            Column(
+                modifier = modifier,
+                verticalArrangement = Arrangement.spacedBy(4.dp)
+            ) {
+                LockNotiTop(
+                    modifier = Modifier
+                        .padding(horizontal = 10.dp)
+                        .padding(top = 4.dp),
+                    drawable = notification.drawable,
+                    appTitle = notification.appTitle,
+                    time = notification.notiTime
+                )
+                LockNotiContent(
+                    modifier = Modifier
+                        .padding(horizontal = 10.dp)
+                        .padding(bottom = 4.dp)
+                        .wrapContentHeight(),
+                    title = notification.title,
+                    content = notification.content
+                )
+            }
+            if (clickableState) {
+                Icon(
+                    modifier = Modifier.align(Alignment.CenterEnd).padding(end = 5.dp),
+                    imageVector = if (expandState)Icons.Filled.ExpandLess else Icons.Filled.ExpandMore,
+                    contentDescription = null
+                )
+            }
+        }
+        AnimatedVisibility(visible = !expandState) {
+            Column {
+                if (notificationList.size == 2) {
+                    MoreNotifcation(modifier = Modifier.padding(horizontal = 15.dp).fillMaxWidth().height(7.dp))
+                } else if (notificationList.size >= 3) {
+                    MoreNotifcation(modifier = Modifier.padding(horizontal = 15.dp).fillMaxWidth().height(7.dp))
+                    MoreNotifcation(modifier = Modifier.padding(horizontal = 35.dp).fillMaxWidth().height(5.dp))
+                }
+            }
+        }
+        AnimatedVisibility(visible = expandState) {
+            Column(
+                modifier = Modifier.padding(top = 4.dp),
+                verticalArrangement = Arrangement.spacedBy(4.dp)
+            ) {
+                for (index in 1 until notificationList.size) {
+                    LockNotiItem(
+                        modifier = Modifier.background(color = Color(0xFFFAFAFA).copy(alpha = 0.95f), shape = RoundedCornerShape(10.dp)).clip(RoundedCornerShape(10.dp)),
+                        notification = notificationList[index]
+                    )
+                }
+            }
+            Spacer(modifier = Modifier.height(4.dp))
+        }
     }
 }
 
+@Composable
+fun MoreNotifcation(
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier = modifier.background(color = Color(0xFFFAFAFA).copy(alpha = 0.9f), shape = RoundedCornerShape(bottomStart = 5.dp, bottomEnd = 5.dp)).clip(shape = RoundedCornerShape(bottomStart = 5.dp, bottomEnd = 5.dp))
+    ) {}
```

Comment:
```suggestion
 val moreNotificationShape = RoundedCornerShape(bottomStart = 5.dp, bottomEnd = 5.dp)
 Row(
        modifier = modifier.background(
            color = Color(0xFFFAFAFA).copy(alpha = 0.9f),
            shape = moreNotificationShape
            )
            .clip(shape = moreNotificationShape )
    ) {}
```

같은 shpae를 쓴다는 의미도 좋고 가독성도 좋아져서 변수화를 해도 좋을 것 같아요!

Reply:
@jaeryo2357 [a63cff8](https://github.com/Knock-Lock/Knock-Lock/pull/75/commits/a63cff80d0bd1a23d6036192457f0c9702a0851f) 반영햇습니다